### PR TITLE
created html, first commit

### PR DESCRIPTION
**Pull request by krissycaron**

### CODE REVIEW

**Comment by krissycaron** [comment]
after having issues trying to pull the inventory.json data from the json file through a function to print o the console, I couldn't figure it out, and was getting very frustrated. I went at it in one file by unlinking the .js files and starting a domHandelr.js and tired to write all the functions within one file to start and go at the problem like we did when we originally did the iife lesson. I am still missing some link in my "getting of the data" I have now been staring at this and trying to peck at it from different angles for 4 days and I need help in understanding the iife concepts again, at least I think that is what my problem is and where I am stuck in wrapping my brain around this problem.

**Comment by krissycaron** [comment]
Hi , If anyone is available today to look at my quiz, I have my focus working, but my cards are reprinting every time I have a keypress, click or anything and I know I am not clearing something out but I could use a little direction, I think its happen in gin my main.js file because on every keypress, I am reprinting the entire set of data, also I have an error on line 37 in my main.js that maybe causing the error.  thank you  :-)

**Comment by zoeames** [comment]
@krissycaron sit with Callan or Lauren on saturday and go over this

**Comment by morecallan** [comment]
@krissycaron -
1. Make sure to look for the line in your click event that is loading your inventory and printing it to the DOM. 
2. I'm still getting errors "can't read innerHTML" of undefined when I click on the cards. Make sure your "selectedCar" is the correct DOM Node for each scenario in the if/else statement.

File: js/eventHandler.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// The second IIFE should augment the original one with a function that creates all of the `eventHandlers` 
+// that you need for the application. Name the function `activateEvents`.
+var CarLot = (function (oldCarLot) {
+	var textBoxNav = document.getElementById("userInputBox");
```
Comment (rousell) [inline js/eventHandler.js:4]: This line of code is not necessary since it is not used in the scope of this iife

**Review by rousell — COMMENTED** [review]
@krissycaron 
nice job! the functionality is all working like it should :) Go ahead and remove your console.log statements, your commented unused code, and create a README for what your project is / how someone could pull down and run your project. Address that last comment that I made, then you should be good to go!

**Comment by rousell** [comment]
@krissycaron 
nice job! the functionality is all working like it should :) Go ahead and remove your console.log statements, your commented unused code, and create a README for what your project is / how someone could pull down and run your project. Address that last comment that I made, then you should be good to go!

**Comment by rousell** [comment]
Hi @krissycaron! I know that you have a lot of assignments, but please try to make these changes so that we can mark this as complete / please let me know if you have any questions!

**Comment by rousell** [comment]
@krissycaron cool looks good! 👍

**Pull request merged**